services/onboarding-service/src/main/java/com/amazon/aws/partners/saasfactory/saasboost/OnboardingService.java

```diff
@@ -835,8 +835,8 @@ public APIGatewayProxyResponseEvent provisionTenant(Map<String, Object> event, C
             try {
                 CreateStackResponse cfnResponse = cfn.createStack(CreateStackRequest.builder()
                         .stackName(stackName)
-                        .onFailure("DO_NOTHING")
-                        .timeoutInMinutes(60)
+                        .onFailure("DO_NOTHING") // This was set to DO_NOTHING to ease debugging of failed stacks. Maybe not appropriate for "production". If we change this we'll have to add a whole bunch of IAM delete permissions to the execution role.
```

Comment:
The problem with ROLLBACK is that we're going to have to add **_lots_** and **_lots_** of IAM permissions to the onboarding microservice execution role to be able to remove the resources. That IAM policy has already given us problems because it's larger than the IAM limits if you spell everything out for least privilege.

Reply:
I hear you and for the time being we can leave it as is.